CODEOWNERS

```diff
@@ -0,0 +1,14 @@
+*  @fjetter
+
+# Dashboard
+distributed/dashboard/*  @jacobtomlinson
+distributed/diagnostics/*  @jacobtomlinson @fjetter
+
+# Deployment tooling
+distributed/deploy/*  @jacobtomlinson
+
+# Ipywidgets
+distributed/widgets/*  @jacobtomlinson
+
+# GPU Support
+distirbuted/diagnostics/nvml.py  @dask/gpu
```

Comment:
GitHub is moaning about this being invalid. I *think* it's because the @dask/gpu team doesn't have wrote permissions to the repo. 

There are a few things we could do here:
- Remove this line and continue with manual pings for review.
- Give the @dask/gpu team write access with the current membership, effectively giving write access to a few more folks.
- Create a second team called @dask/gpu-reviewers containing the members of @dask/gpu who have write access today.

Reply:
While this is technically covered by our governance docs, we do not have any team with write permission set up anywhere as far as I know.
Historically, we've been also very conservative when it comes to handing out write permissions for this repo. Just for the sake of having an easier way to get notifications does not meet the threshold form my perspective.

There is also a fourth option of not using a team but just putting the few individuals with write permissions who are owning this in here.

Reply:
> Just for the sake of having an easier way to get notifications does not meet the threshold form my perspective.

Agreed. I think this would only be valuable if it were to be used to make folks responsible for reviewing (my second point).

> There is also a fourth option of not using a team but just putting the few individuals with write permissions who are owning this in here.

True, I think I overthought this. Perhaps this should just be me and @quasiben for now?

Reply:
> True, I think I overthought this. Perhaps this should just be me and @quasiben for now?

Works for me if @quasiben is OK with it (General thought and hopefully obvious, we should not add anybody here w/out their approval)

Reply:
> General thought and hopefully obvious, we should not add anybody here w/out their approval

Yeah absolutely. This should be opt in for sure.